### DISCUSSION TITLE
Prosperity cap

### DISCUSSION THREAD

**Discussion by mathieumahe** [Keriew/augustus#341] [Ideas]
Hi

First, I want to thank you for the amazing work. I'm so glad to be able to play this game again, after 20 years, with all those new improvements!

That being said, I have struggled (once more..) on prosperity during my games. I found this [article](https://caesar3.heavengames.com/cgi-bin/caeforumscgi/display.cgi?action=ct&f=2,1489,840,all) online, saying that the game have an internal limitation on your prosperity rating.

Do you think it would be possible to display this variable somewhere so we can better understand what is happening in our cities?

Thanks in advance!

**Comment by Keriew** [comment]
Implemented in 0f99c38, will be included in the next release.

**Reply by mathieumahe** [reply]
That's awesome! Thanks

**Reply by brainwad** [reply]
BTW, isn't this already shown on the Population - Society advisor screen? It says, e.g.  "Housing prosperity rating is 115"

**Reply by mathieumahe** [reply]
I just checked and you are right. TIL
Sorry for the inconvenience
